flax/nnx/transforms/compilation.py

```diff
@@ -208,12 +210,14 @@ def jit(
           determine the output shardings.
 
       The size of every dimension has to be a multiple of the total number of
-      resources assigned to it. This is similar to pjit's in_shardings.
-    out_shardings: Like ``in_shardings``, but specifies resource
-      assignment for function outputs. This is similar to pjit's
-      out_shardings.
-
-      The ``out_shardings`` argument is optional. If not specified, :py:func:`jax.jit`
+      resources assigned to it. This is similar to `pjit <https://jax.readthedocs.io/en/latest/jax.experimental.pjit.html>`_
```

Comment:
The entire docstring might be more `jax.jit`-related. Should we adapt it to `flax.nnx.jit`? @cgarciae @IvyZX